test/e2e/cypress/integration/hana_cluster_details.js

```diff
@@ -69,6 +69,18 @@ context('HANA database details', () => {
         .next()
         .contains(availableHanaCluster.cibLastWritten);
     });
+
+    it('should have the check overview component with passing checks', () => {
```

Comment:
Could we maybe check if the value of each of the entries is correct? To be a more robust test

Reply:
We can but the effort is non trivial, we need a very robust and predictable fixture, Maybe we can schedule this type of testing and fixture creating for the future?